magento/magento2
# #22084: Remove hard dependency on Magento_Wishlit with Magento_Captcha

### Description (*)
Another Pull Request added a hard dependency between Magento_Captcha and Magento_Wishlist

### Fixed Issues (if relevant)
1. magento/magento2#22084: [2.3.1] Wishlist module shouldn't have a hard dependency on Captcha module

### Manual testing scenarios (*)
1. Magento 2.3.* with sample data installed
2. Enable captcha on wishlist sharing form
3. Add product to wishlist
4. Try to share wishlist, you should be able to see a captcha and only submit if captcha is correct
5. Remove captcha from wishlist sharing form (but keep module enabled)
6. Try to share wishlist, you should be able to submit normally without captcha
7. Disable Captcha module
8. Try to share wishlist, you should be able to submit normally without captcha

### Questions or comments
Please let me know if I need to update any Tests

### Contribution checklist (*)
